test/pub_test.dart

```diff
@@ -149,7 +139,7 @@ dev_dependencies:
 PSDependency findDependency(PSDependencyList deps, {String name}) =>
     deps.firstWhere((dep) => dep.name.text == name, orElse: () => null);
 
-testDepListContains(
+void testDepListContains(
```

Comment:
Clarity for future maintainers.

Since omitting a return annotation implies dynamic, it's super clear that there is *no* expectation that this function could/should return a value.

We spend much more time reading code than writing. Explicit intent FTW! :metal: